include/zephyr/drivers/dma/dma_smartbond.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2023 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef DMA_SMARTBOND_H_
+#define DMA_SMARTBOND_H_
+
+/*
+ * DMA channel priority level. The smaller the value the lower the priority granted to a channel
+ * when two or more channels request the bus at the same time. For channels of same priority an
+ * inherent mechanism is applied in which the lower the channel number the higher the priority.
+ */
+enum dma_smartbond_channel_prio {
```

Comment:
prio, channel, burst len and bus width are generic numbers and don't really need special defines

mux is the only one that is hw specific

Reply:
Enumerations are used so the code is more human readable. These are also used by the driver implementation (instead of using plain numbers). Should these structures be removed?

Reply:
so those can be private for the driver, if needed.
I don't see any other driver exposing public symbols for priorities, channels etc. since they are just generic numbers. mux is the only one that is hw specific.

Reply:
Generic enumerators were transferred to `dma_smatbond.c`